src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -443,23 +492,37 @@ private boolean isHomePage(String path) {
   }
 
   private void styleHomepage() {
-    final int windowHeight = $(window).height();
-    int sectionHeight = $("#letsbegin").height();
+    final int windowHeight = window.innerHeight;
+    int sectionHeight = Js.<HTMLElement>uncheckedCast(document.querySelector("#letsbegin")).offsetHeight;
 
     if (windowHeight > sectionHeight) {
-      $(".home section").each(new Function() {
-        @Override
-        public void f() {
-          $(this)
-              .css("height", windowHeight + "px")
-              .css("padding", "0");
-          GQuery container = $(this).find(".container");
-          container.css("padding-top", (windowHeight - container.height()) / 2 + "px");
-        }
+      forEach(".home section", element -> {
+        element.style.setProperty("height", windowHeight + "px");
+        element.style.setProperty("padding", "0");
+        HTMLElement container = Js.uncheckedCast(element.querySelector(".container"));
+        container.style.setProperty("padding-top", (windowHeight - container.offsetHeight) / 2 + "px");
       });
     }
   }
 
+  private void forEach(String selector, Consumer<HTMLElement> consumer) {
+    document.querySelectorAll(selector).forEach((element, index, parent) -> {
+      consumer.accept(Js.uncheckedCast(element));
+      return null;
+    });
+  }
+
+  private void forEachChild(Element parent, Predicate<HTMLElement> filter, Consumer<HTMLElement> consumer) {
+    parent.childNodes.forEach((element, index, p) -> {
+      if (element.nodeType == Node.ELEMENT_NODE) {
```

Comment:
Instead of childNodes and then testing if each is an element, I think you just want to use [parent.children](https://developer.mozilla.org/en-US/docs/Web/API/Element/children), which will only return elements to begin with.

(again, can wait until a later PR, it doesn't impact correctness, and is a more literal translation of what we started with)

Reply:
Also note that `element instanceof HTMLElement` would achieve the same effect (and would make the two casts below unnecessary... esp with java 17 almost here for GWT).

Reply:
I kept instanceof, if we want to use `children` we might want to bump Elemental and therefore Java (to 11).